lib/services/alexa/request/lifecycle/response.ts

```diff
@@ -60,6 +61,8 @@ export const responseGenerator = (utils: typeof utilsObj) => async (runtime: Ale
     payload: response,
     sessionid: input.requestEnvelope.session?.sessionId,
     metadata: runtime.getFinalState(),
+    timestamp: new Date(),
```

Comment:
Shouldn't this be serialized with a toString()?

Reply:
gets serialized once when building request body

https://github.com/voiceflow/alexa-runtime/blob/795cea0df66c5b5ea54d3f97d66710fff520c3d0/lib/clients/analytics.ts#L100

Reply:
yup!